### PR TITLE
added swablu to manage

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by JensTimmerman** [comment]
The grimer class needs the same additions like the swablu class

File: config/manage_defaults.cfg [hunk 1]
```diff
@@ -12,6 +12,7 @@ IMM_USER_PHANPY = USERID
 IMM_USER_GOLETT = USERID
 IMM_USER_SHUPPET = testuser
 IMM_USER_SWALOT = root
+IMM_USER_SWABLU = root
```
Comment (wdpypere) [inline config/manage_defaults.cfg]: swablu does not have IMM's? actually, nobody since haunter and gastly had IMM's. (IMM = IBM, iDRAC=Dell, iLO=HPe)

Comment (JensTimmerman) [reply config/manage_defaults.cfg]: Yeah, BMC, IMM, iDRAC and iLO are all called IMM in manage.

File: config/manage_defaults.cfg [hunk 2]
```diff
@@ -12,6 +12,7 @@ IMM_USER_PHANPY = USERID
 IMM_USER_GOLETT = USERID
```
[the remark this answers is not in the document]
Comment (JensTimmerman) [reply config/manage_defaults.cfg]: This is a defaults file, this value is set correctly in `/etc/manage.cfg` by quattor.

**Pull request merged**